File: PYTHON/aksesoris.py
from petshop import petshop

class aksesoris(petshop):

    # ATRIBUT PRIVATE
    __jenis = ""
    __bahan = ""
    __warna = ""

    # CONSTRUCTOR
    def __init__(self, id, nama_produk, harga_produk, stok_produk, jenis="", bahan="", warna=""):
        super().__init__(id, nama_produk, harga_produk, stok_produk)
        self.__jenis = jenis
        self.__bahan = bahan
        self.__warna = warna

    # GETTER
    def get_jenis(self):
        return self.__jenis

    def get_bahan(self):
        return self.__bahan

    def get_warna(self):
        return self.__warna

    # SETTER
    def set_jenis(self, jenis):
        self.__jenis = jenis

    def set_nama_produk(self, nama_produk):
        self.__nama_produk = nama_produk

    def set_harga_produk(self, harga_produk):
        self.__harga_produk = harga_produk

    def set_stok_produk(self, stok_produk):
        self.__stok_produk = stok_produk

File: JAVA/aksesoris.java
import java.util.ArrayList;

public class aksesoris extends petshop{
    private String jenis;
    private String bahan;
    private String warna;
    
        // CONSTRUCTOR
        public aksesoris(){
            this.jenis = "";
            this.bahan = "";
            this.warna = "";
        }

        // CONSTRUCTOR dengan parameter
        public aksesoris(String id, String nama_produk, int harga_produk, int stok_produk, String jenis, String bahan, String warna){
            super(id, nama_produk, harga_produk, stok_produk); 
            this.jenis = jenis;
            this.bahan = bahan;
            this.warna = warna;
        }

        // GETTER
        public String get_jenis(){
            return this.jenis; 
        }
        public String get_bahan(){
            return this.bahan; 
        }
        public String get_warna(){
            return this.warna; 
        }
        
        // SETTER
        public void set_jenis(String jenis){
            this.jenis = jenis;
        }
        public void set_bahan(String bahan){
            this.bahan = bahan;
        }
        public void set_warna(String warna){
            this.warna = warna;
        }
};

